app/api/i18n/specs/defaultTranslations.spec.ts

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable node/no-restricted-import */
+import { readdir } from 'fs/promises';
+import request from 'supertest';
+import waitForExpect from 'wait-for-expect';
+
+import { validateFormat, ValidateFormatError } from 'api/csv/csv';
+import { expectThrow } from 'api/utils/jestHelpers';
+import { DBFixture } from 'api/utils/testing_db';
+import { iosocket, setUpApp } from 'api/utils/testingRoutes';
+import { testingEnvironment } from 'api/utils/testingEnvironment';
+import { UserRole } from 'shared/types/userSchema';
+
+import { DefaultTranslations } from '../defaultTranslations';
+import i18nRoutes from '../routes';
+import translations from '../translations';
+
+const TRANSLATION_FILES_DIR = DefaultTranslations.CONTENTS_DIRECTORY;
+
+type defaulTranslationInfo = {
+  key: string;
+  longName: string;
+};
+
+const EXPECTED_DEFAULT_TRANSLATIONS: defaulTranslationInfo[] = [
+  {
+    key: 'ar',
+    longName: 'Arabic',
+  },
+  {
+    key: 'en',
+    longName: 'English',
+  },
+  {
+    key: 'es',
+    longName: 'Spanish',
+  },
+  {
+    key: 'fr',
+    longName: 'French',
+  },
+  {
+    key: 'ko',
+    longName: 'Korean',
+  },
+  {
+    key: 'my',
+    longName: 'Burmese',
+  },
+  {
+    key: 'ru',
+    longName: 'Russian',
+  },
+  {
+    key: 'th',
+    longName: 'Thai',
+  },
+  {
+    key: 'tr',
+    longName: 'Turkish',
+  },
+];
+const expectedFileNames = new Set(EXPECTED_DEFAULT_TRANSLATIONS.map(({ key }) => `${key}.csv`));
+
+const fixtures: DBFixture = {
+  settings: [
+    {
+      languages: [
+        {
+          key: 'en',
+          label: 'English',
+          default: true,
+        },
+        {
+          key: 'es',
+          label: 'Spanish',
+        },
+      ],
+    },
+  ],
+};
+
+afterAll(async () => {
+  await testingEnvironment.tearDown();
+});
+
+describe('translations.importPredefined()', () => {
+  beforeAll(async () => {
+    DefaultTranslations.CONTENTS_DIRECTORY =
+      './app/api/i18n/specs/test_contents/filesForDefaultTranslations';
+    await testingEnvironment.setUp(fixtures);
+  });
+
+  afterAll(async () => {
+    DefaultTranslations.CONTENTS_DIRECTORY = TRANSLATION_FILES_DIR;
+  });
+
+  it.each([{ key: 'en' }, { key: 'ar' }])(
+    'should expect the file to have 2 columns',
+    async ({ key }) => {
+      await expectThrow(
+        async () => translations.importPredefined(key),
+        ValidateFormatError,
+        'Expected 2 columns, but found 3.'
+      );
```

Comment:
There is a builtin solution for this, or maybe you tried and it does not work for you ? Something like this.
```typescript
await expect(async () => translations.importPredefined(key)).rejects.toEqual(
  new ValidateFormatError('Entity deletion error !')
);

```